deep-learning/_posts/2017-08-17-S3Pool.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: S3Pool - Pooling with stochastic Spatial Sampling
+tags: deep-learning pooling classification
+cite:
+    authors: "S. Zhai, H. Wu, A. Kumar, Y. Cheng, Y. Lu, Z. Zhang, R. Feris "
+    title:   "S3Pool : Pooling with stochastic Spatial Sampling"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1611.05138.pdf"
+---
+
+# Idea
+   New kind of pooling aiming to learn from the information that is discarded by classical pooling. The image is spatially partioned and the rows and columns of a pooling grid are non-uniformly and stochastically chosen. It acts as a strong regularizer equivalent to implicit data augmentation because the network will see a different distorsion of the feature maps at each pass.  
+   
+# Method
+1. Partition the image (h,w) into strips of  size g (both vertically and horizontally)
+2. Randomly select g/s rows and columns from each strip to obtain the downsampling grid
+→ the resulting feature map is of siwe h/s, w/s (like a pooling with a window of size s)
+→ the pooling is likely to be non-uniform (different at each pass for the same image)
+3. At test time, it is replaced either by standard max pooling or average pooling
+
+# Application
+Tested on : CIFAR10-100 and STL-10
+With : ResNet 
+
+This method performs better than dropout and classical stochastic pooling both with and without data augmentation.
+
+! The training time goes up by about 10-15% !
+
+[code](https://github.com/Shuangfei/s3pool).
+
+Downsampling : regular and stochastic spatial with different grid sizes
```

Comment:
You need to add two newline after l.32, because the output want to align the sentence next to the figure. That make the sentence which has space between word more like each word is separated by a tabulation.

```markdown

Downsampling : regular and stochastic spatial with different grid sizes


![](/deep-learning/images/S3Pool/distorsion.png)
```

I'll do the same for the image below (pooling stategies).